src/dropbox.js

```diff
@@ -69,9 +69,9 @@ export default class Dropbox {
     this.selectAdmin = options.selectAdmin;
     this.pathRoot = options.pathRoot;
 
-    this.domain = options.domain;
-    this.domainDelimiter = options.domainDelimiter;
-    this.customHeaders = options.customHeaders;
+    this.domain = options.domain || this.auth.domain;
```

Comment:
If auth is undefined will this error?

Reply:
Oh right. I see what you're saying now. this.auth could be undefined due to this [code](https://github.com/dropbox/dropbox-sdk-js/blob/main/src/dropbox.js#L62)

Reply:
Oh wait. Actually, nvm! `this.auth` will never be undefined since `this.auth` only gets set if `options.auth` exists, otherwise it'll instantiate a new auth object